conda/recipes/cucim/meta.yaml

```
@@ -43,7 +43,7 @@ requirements:
     - cudatoolkit ={{ cuda_version }}
     - cupy >=10,<12.0.0a0
     - libcucim ={{ version }}
-    - numpy >=1.21.3
+    - numpy >=1.21.3,<1.24
```

Discussed with @ajschmidt8 offline. This was solving to `1.24.1` in the `host`, which meant that the runtime `pin_compatible` was something like `>=1.24` -- which is incompatible with cudf.

The solution is that `host` requirements should build against the oldest supported NumPy. Then the `pin_compatible` won't conflict with cudf's requirements.
```suggestion
    - numpy =1.21.3
```